src/plugins/condition/components/inspector/ConditionalStylesView.vue

```diff
@@ -1,14 +1,43 @@
 <template>
-<div>Conditional Styles inspector view</div>
+<div class="c-properties__row--span-all">
+    <div class="controls">
+        <div class="preview">ABC</div>
+        <button class="c-icon-button--swatched icon-paint-bucket"
+                data-style-prop="backgroundColor"
+                @click="applyStyle"
+        ></button>
+        <button class="c-icon-button--swatched icon-line-horz"
+                data-style-prop="borderColor"
+                @click="applyStyle"
+        ></button>
+        <button class="c-icon-button--swatched icon-font"
+                data-style-prop="color"
+                @click="applyStyle"
+        ></button>
+    </div>
+</div>
 </template>
 
 <script>
 
 export default {
-    components: {
-    },
+    name: 'ConditionalStylesView',
     inject: [
         'openmct'
-    ]
+    ],
+    data() {
+        return {
+            backgroundColor: '#666',
```

Comment:
Please talk to @charlesh88 about getting the defaults for all the domain objects we have. We should probably define these either:
a) In a constants file where we map the type of object to it's defaults OR
b) In the configuration of the domain object - the issue with this is that not all the domain objects have configurations so we might have to write a migration script - @akhenry might have thoughts on this.

Reply:
Will discuss with @charlesh88 and bring in @akhenry if needed.